working/2364 - primary constructors/feature-specification.md

```diff
@@ -361,14 +397,28 @@ constants *(holding parameter default values)*.
 Where no processing is mentioned below, _D2_ is identical to _D_. Changes
 occur as follows:
 
+Assume that `p` is an optional formal parameter in _D_ that does not have a
+declared type, but it does have a default value whose static type in the
```

Comment:
Should probably not apply to initializing formals or super-parmeters (if we allow those) because they get their type from their target instead.

Reply:
Good catch! Done.